packages/animations/example/lib/main.dart

```diff
@@ -45,39 +45,39 @@ class _TransitionsHomePageState extends State<_TransitionsHomePage> {
             child: ListView(
               children: <Widget>[
                 _TransitionListTile(
-                  title: 'Shared Axis',
-                  subtitle: 'SharedAxisTransition',
+                  title: 'Container Transform',
```

Comment:
Can you change this to "Container transform" (lowercase T)

Same for the other demos, only the first word should be capitalized.

Reply:
Can you also check the app bar titles of the actual demos to match this capitalization?